proposals/description/buyback_newpool.ts

```diff
@@ -0,0 +1,51 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const buyback_newpool: ProposalDescription = {
+  title: 'Update the TRIBE buyback Balancer LBP',
+  commands: [
+    {
+      target: 'pcvEquityMinter',
+      values: '0',
+      method: 'setTarget(address)',
+      arguments: ['{noFeeFeiTribeLBPSwapper}'],
+      description: 'Set PCV Equity Minter target to new Buyback swapper'
+    },
+    {
+      target: 'feiTribeLBPSwapper',
+      values: '0',
+      method: 'exitPool(address)',
+      arguments: ['{noFeeFeiTribeLBPSwapper}'],
+      description: 'Exit buyback liquidity from old swapper to new swapper'
+    },
+    {
+      target: 'fei',
+      values: '0',
+      method: 'mint(address,uint256)',
+      arguments: ['{noFeeFeiTribeLBPSwapper}', '4000000000000000000000000'],
+      description: "Mint 4m FEI for missed buybacks and this week's buybacks"
+    },
+    {
+      target: 'pcvEquityMinter',
+      values: '0',
+      method: 'unpause()',
+      arguments: [],
+      description: 'Unpause the PCV Equity Minter'
+    },
+    {
+      target: 'noFeeFeiTribeLBPSwapper',
+      values: '0',
+      method: 'forceSwap()',
+      arguments: [],
+      description: 'Re-start the buybacks'
```

Comment:
Is there any access control cleanup or Collateralization oracle items to do? I think we need a new lens.

Reply:
I don't think we have access control to change, the only special thing was to give `SWAP_ADMIN_ROLE` to the pcvEquityMinter, which it still has

I'll deploy a new lens & swap the old one for the new one in the CR oracle